3/README.md

```diff
@@ -1,16 +1,15 @@
 ```
 shortname: 3/UPSERT-VALIDATORS
-name: Dynamically add/remove validators at runtime (Insecure version)
+name: Dynamically add/update/remove validators at runtime
 type: standard
 status: raw
```

Comment:
Status is `raw` here, but `stable` in the main `README.md` table. My suggestion is to keep this proposal `draft` for now, and mark it `stable` when BigchainDB 2.0 is not alpha anymore, what do you think?

Reply:
Oops, good catch! I'll fix that.